test/e2e/common/types.py

```diff
@@ -1,2 +1,3 @@
+GROUP_RESOURCE_PLURAL = 'groups'
 ROLE_RESOURCE_PLURAL = 'roles'
 POLICY_RESOURCE_PLURAL = 'policies'
```

Comment:
I don't really care for this file. I think it'd be better to just have these defined as consts in each of the test files, then import them as necessary into other test files